arctic-report-card/ARCRC.ttl

```diff
@@ -232,508 +262,647 @@ Find current and past assessments of the Arctic Report Card Vital Signs by visit
+odo:ARCRC_00000515 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000042 ;
+                   dc:creator <http://orcid.org/0000-0003-0632-7576> ;
+                   dc:date "2020-12-01T01:28:06Z"^^xsd:dateTime ;
+                   dc:description "TI (time-integrated) NDVI is the sum of the biweekly NDVI values for the growing season and is correlated with the total aboveground vegetation productivity." ;
+                   rdfs:label "TI-NDVI"@en ;
+                   skos:altLabel "TI (time-integrated) NDVI" .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000015
+odo:ARCRC_00000015 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000512 ;
+                   obo:IAO_0000115 "Subsea permafrost in the Arctic is generally relict terrestrial permafrost, inundated after the last glaciation and now degrading under the overlying shelf sea." ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   rdfs:label "Subsea Permafrost Indicator"@en ;
+                   skos:broadMatch "http://www.purl.dataone.org/odo/ECSO_00010130" .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000016
+odo:ARCRC_00000016 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000513 ;
+                   obo:IAO_0000115 "Applying and centering Indigenous perspectives and observations of Arctic change" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   rdfs:label "Working with Communities to Observe the Arctic"@en .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000017
+odo:ARCRC_00000017 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000513 ;
+                   obo:IAO_0000115 "Here, we refer to “western Alaska” salmon as those spawning in watersheds that drain into the eastern Bering Sea." ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   rdfs:label "Western Alaska Salmon"@en ;
+                   skos:relatedMatch "http://purl.dataone.org/odo/SALMON_00000215" .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000018
+odo:ARCRC_00000018 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "Any product of the condensation of atmospheric water vapor that falls under gravity" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Vital Sign, 'Precipitation.'" ;
+                   rdfs:label "precipitation"@en ;
+                   owl:sameAs "http://purl.obolibrary.org/obo/ENVO_01000703" .
```

Comment:
Probably should use `owl:equivalentClass`. Let's discuss the difference.

Reply:
Changed it to equivalentClass, but am interested in the difference between the two